docs/demo.md

```
@@ -0,0 +1,170 @@
+<div align="center">
+
+# <img src="https://citadelfoss.xyz/coinswap-bw.svg" alt="Coinswap" height="36" style="vertical-align: middle;"/> Coinswap Live Demo
+
+### Prerequisites & Setup Guide
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)